sdf.md

```diff
@@ -263,29 +298,46 @@ Class:
 : Abstract term for the information that is contained in groups
   identified by a Class Name Keyword.
 
-Property:
-: An affordance that can potentially be used to read, write, and/or
-  observe state (current/stored information) on a Grouping.
-  (Note that Entries are often called
-  properties in other environments; in this document, the term
-  Property is specifically reserved for affordances, even if the map
-  key "properties" might be imported from a data definition language
-  with the other semantics.)
+Quality:
+: A metadata item in a definition or declaration which says something
+  about that definition or declaration.  A quality is represented in
+  SDF as an entry in a JSON map (JSON object) that serves as a definition
+  or declaration.
+  (The term "Quality" is used because another popular term,
+  "Property", already has a different meaning.)
 
-Action:
-: An affordance that can potentially be used to perform a named operation on a Grouping.
+Definition:
+: An entry in a Definition Group.
```

Comment:
Should Definition Group maybe also become its own term? Currently, it is only defined within Group, so you need to look into the description to find it.